src/common.build.exe.or.dll.props

```diff
@@ -6,7 +6,7 @@
     </ClCompile>
     <Link>
       <ProgramDatabaseFile>$(OutDir)$(TargetName)FullPDB.pdb</ProgramDatabaseFile>
-      <AdditionalDependencies>dwrite.lib;dxgi.lib;d2d1.lib;d3d11.lib;shcore.lib;uxtheme.lib;dwmapi.lib;winmm.lib;pathcch.lib;propsys.lib;uiautomationcore.lib;Shlwapi.lib;ntdll.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>onecore_apiset.lib;dwrite.lib;dxgi.lib;d2d1.lib;d3d11.lib;shcore.lib;uxtheme.lib;dwmapi.lib;winmm.lib;pathcch.lib;propsys.lib;uiautomationcore.lib;Shlwapi.lib;ntdll.lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
dwmapi, winmm, shlwapi are all in different forms on OneCore editions than on desktop, so linking those libs explicitly (instead of via umbrella) is probably not good.

pathcch isn't there at all... so maybe the API we're using through that is available in a different way?

Reply:
Thanks to ordering, they’re not linked once onecore is hit. Pathcch doesn’t seem to have analogues unless we switch wholly to std::filesystem, which we probably should do but is out of scope for link line changes

Reply:
file as issue, help-wanted?

Reply:
dwmapi and winmm were both still required. I'll revisit the link line later.